content/src/Server.ts

```diff
@@ -67,6 +87,7 @@ export class Server {
       Metrics.initialize()
     }
 
+    this.app.options('*', cors(corsOptions))
```

Comment:
Line 80 already registers cors for all methods, this may be unnecesary

Reply:
Line 80 registers cors for all methods and paths that are configured later, but the OPTIONS method is never enabled. I could modify the registerRoute method, to also register OPTIONS method for each path, so we can avoid answering any path if that seems better.

Reply:
https://github.com/expressjs/cors/blob/master/lib/index.js#L163-L171 

The `cors` package manages the `OPTIONS` method internally